src/main/bindings/c/bindings.h

```diff
@@ -293,6 +293,11 @@ void statusLogger_updateEmuTime(const struct StatusLogger_ShadowStatusBarState *
 void statusLogger_updateNumFailedProcesses(const struct StatusLogger_ShadowStatusBarState *status_logger,
                                            uint32_t num_failed_processes);
 
+// Get the backtrace. This function is slow. The string must be freed using `free_backtrace()`.
```

Comment:
`backtrace_free()`?

Reply:
Oops nice catch! I'll fix that in a follow-up PR.